### PR TITLE
[MB-2125] Added scripts to update version and set up for tests.

### PR DESCRIPTION
**Pull request by hcrowell**
Improve our Cordova release process:
* Added script to update the `urbanairship-cordova` version
* Added script to set up android and iOS testing.

Testing:
* Ran scripts to update version and set up android & iOS testing. Received pushes on iPhone and Nexus 5x

### CODE REVIEW

**Comment by crow** [comment]
🍔  looks good

File: .gitignore [hunk 1]
```diff
@@ -7,4 +7,6 @@ src/android/.idea/
 src/android/android.iml
 src/android/local.properties
 out/*
-node_modules/*
\ No newline at end of file
+node_modules/*
+test/config_sample.xml
+test/test/*
```
Comment (rlepinski) [inline .gitignore]: test/test/*?

File: test/config_sample.xml [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        <allow-intent href="market:*" />
+        <!-- UA Android SDK requires minimum sdk version to be 16 -->
+        <preference name="android-minSdkVersion" value="16" />
+    </platform>
+    <platform name="ios">
+        <allow-intent href="itms:*" />
+        <allow-intent href="itms-apps:*" />
+    </platform>
+<!-- Urban Airship app credentials -->
+<preference name="com.urbanairship.production_app_key" value="appKey" />
```
Comment (rlepinski) [inline test/config_sample.xml]: APP_KEY_PLACEHOLDER

File: test/config_sample.xml [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+<preference name="com.urbanairship.production_app_secret" value="appSecret" />
+<preference name="com.urbanairship.development_app_key" value="appKey" />
+<preference name="com.urbanairship.development_app_secret" value="appSecret" />
+<!-- Required for Android. -->
+<preference name="com.urbanairship.gcm_sender" value="12345467890" />
```
Comment (rlepinski) [inline test/config_sample.xml]: GCM_SENDER_ID_PLACEHOLDER

**Comment by rlepinski** [comment]
I have had issues in the past having a test project in the same repo as the plugin. Lets add a argument for a path to generate the test project. `./create_sample ../test`

File: test/test_sample.sh [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+cp ../../Example/js/* www/js
+
+# add the device plugin
+cordova plugin add cordova-plugin-device
+
+# set up iOS
+cordova platform add ios
+cd platforms/ios/
+pod update
+cd ../..
```
Comment (madelgi) [inline test/test_sample.sh]: maybe `pod update --project-directory=$TEST_DIR/platforms/ios` instead?

File: test/test_sample.sh [hunk 5]
```diff
@@ -0,0 +1,45 @@
+#!/bin/bash -ex
+
+# Set up script for iOS and android testing.
+# Follow the steps below.
+# 1. Add the UA credentials to the `config_sample.xml` file in the test directory and save.
+# 2. Run the script with the command `./test_sample.sh `
+# 3. Build the platform you want to test (see comments below).
+
+# keep cordova up to date
+npm install cordova -g
```
Comment (hcrowell) [inline test/test_sample.sh]: thanks for your suggestion, but i think its best to keep cordova up to date

**Comment by madelgi** [comment]
⛵️

**Pull request merged**